### DISCUSSION TITLE
make GAP string available in quippy

### DISCUSSION THREAD

**Discussion by richardjana** [libAtoms/QUIP#562] [Ideas]
After importing a GAP potential via quippy (i.e. `pot = quippy.potential.Potential(param_filename='GAP.xml')`), I thought it could be useful to access the GAP xml_label (e.g. `"Potential xml_label=GAP_2023_3_8_120_17_55_52_832"`). An example use case could be to write it as a comment into result files generated with the GAP.

A good place for this could be the "name" property of the potential object, which currently is not very helpful: `'potential'`. I assume this would be easy to implement, as the xml file containing the GAP has to be read anyways?

**Comment by jameskermode** [comment]
Good idea. This information is likely available already within the underlying `Potential._quip_potential` object, which is an instance of the the auto generated wrapper class. A PR to set Potential.name to the type of potential (eg IP GAP) and/or the XML label would be welcome.
